include/xtensor-python/pynative_casters.hpp

```diff
@@ -0,0 +1,52 @@
+/***************************************************************************
+* Copyright (c) Wolf Vollprecht, Johan Mabille and Sylvain Corlay          *
+* Copyright (c) QuantStack                                                 *
+*                                                                          *
+* Distributed under the terms of the BSD 3-Clause License.                 *
+*                                                                          *
+* The full license is in the file LICENSE, distributed with this software. *
+****************************************************************************/
+
+#ifndef PYNATIVE_CASTERS_HPP
+#define PYNATIVE_CASTERS_HPP
```

Comment:
This file should be added to the header list in `CMakeLists.txt` so that it gets installed when installing `xtensor-python`.